data_registry/process_manager/task/process.py

```diff
@@ -76,6 +76,15 @@ def get_status(self):
             self.job.ocid_prefix = meta.get("ocid_prefix") or ""
 
         self.job.context["process_id_pelican"] = compiled_collection["id"]
+
+        response = self.request(
+            "GET",
+            url_for_collection(original_collection["id"], "notes"),
+            error_message=f"Unable to get notes of collection {original_collection['id']}",
+        )
+
+        self.job.process_notes = response.json()
```

Comment:
```suggestion
        self.job.process_notes = self.request(
            "GET",
            url_for_collection(original_collection["id"], "notes"),
            error_message=f"Unable to get notes of collection {original_collection['id']}",
        ).json()
```

Reply:
Can you update the other two requests to avoid the temporary variable in the same way (because it suggests to the reader that it is used again elsewhere)?